frontend/src/app/features/work-packages/components/wp-card-view/event-handler/right-click-handler.ts

```diff
@@ -21,24 +22,24 @@ export class CardRightClickHandler implements CardEventHandler {
     card:WorkPackageCardViewComponent) {
   }
 
-  public get EVENT() {
-    return 'contextmenu.cardView.rightclick';
+  public get EVENT():EventType {
+    return 'contextmenu'; // N.B.: contextmenu is not supported by Safari on iOS.
```

Comment:
What does that comment mean? Is right click not supported on iOS Safari any more?